index.html

```diff
@@ -230,15 +329,89 @@
 <script src="bower_components/angular/angular.min.js"></script>
 <script src="vendor/mm-foundation-tpls-0.5.1.min.js"></script>
 <script src="dist/js/jsflap.js"></script>
-<script>
+<!-- <script>
   (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
   (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
   })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-  ga('create', 'UA-67654300-1', 'auto');
-  ga('send', 'pageview');
+  //ga('create', 'UA-67654300-1', 'auto');
+  //ga('send', 'pageview');
+
+</script> -->
+<!-- <script type="text/javascript">
+function parseGraph(contents) {
+    // Currently hard coded to parse a JSON file. Will later be remade to intuitively parse files
```

Comment:
Ok, you are making this ***significantly*** harder than it needs to be, there are built-in methods `JSON.parse` which is all you need for this seriously, you don't need to do any string parsing although I do applaud the effort here.